channels/in.m3u

```diff
@@ -1,4 +1,6 @@
 #EXTM3U
+#EXTINF:-1 tvg-id="AlJazeeraEnglishHD.in" tvg-country="IN" tvg-language="English" tvg-logo="https://cdn.sat.tv/wp-content/uploads/2016/05/AlJazeeraEng.png" group-title="News",AlJazeera English HD
+http://14.199.164.20:4001/play/a0wa/index.m3u8
```

Comment:
Quatar channel, no need to stress HD in name

Reply:
Yup, this channel originates from Qatar (qa), we already have that one by the way.

Reply:
can you guys remove this channel add the rest?

Reply:
Deleted it.